src/components/legend/draw.js

```diff
@@ -455,6 +457,13 @@ function handleClick(g, gd, numClicks) {
         tracei,
         newVisible;
 
+
+    if(numClicks === 1 && SHOWISOLATETIP && gd.data && gd._context.showTips) {
```

Comment:
We don't need `gd.data` here - `SHOWZOOMOUTTIP` needed it because you can't autoscale a plot with nothing on it, but there wouldn't be a legend anyway without data!